cas-server-core-api-services/src/main/java/org/apereo/cas/services/ServiceRegistryDao.java

```diff
@@ -41,4 +42,19 @@
      * @return the registered service
      */
     RegisteredService findServiceById(long id);
+
+    /**
+     * Return number of records held in this service registry.
+     *
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Empty line should be followed by <p> tag on the next line.](https://www.codacy.com/app/mmoayyed/cas/file/2579583011/issues/source?bid=3256115&fileBranchId=3297474#l48)